### PR TITLE
Aula (login) metrics

### PR DESCRIPTION
**Pull request by andorp**

### CODE REVIEW

File: src/Frontend.hs [hunk 1]
```diff
@@ -57,24 +58,26 @@ runFrontend :: Config -> IO ()
 runFrontend cfg = do
     log <- logDaemon (cfg ^. logging)
     void $ log ^. start
-    waiMetrics <- startEKG cfg
-    runFrontendWithLogger cfg (log ^. msgDaemonSend) waiMetrics
+    metrics <- startEKG cfg
+    runFrontendWithLogger cfg (log ^. msgDaemonSend) metrics
 
-startEKG :: Config -> IO (Maybe EKG.WaiMetrics)
+startEKG :: Config -> IO (Maybe (EKG.WaiMetrics, AulaMetrics))
```
Comment (fisx) [inline src/Frontend.hs]: would it make sense to add `WaiMetrics` as a field to `AulaMetrics`?

Comment (fisx) [inline src/Frontend.hs]: ah, that would have the disadvantage that it's being carried around inside the `Action`.

**Comment by fisx** [comment]
LGTM!

File: src/Frontend.hs (hunk 1 above)
Comment (andorp) [inline src/Frontend.hs]: And never used, or used in an inappropriate way.

**Pull request merged**